GitSearch/Application/SearchResult/SearchResultCell.swift

```diff
@@ -112,7 +34,93 @@ class SearchResultCell: UICollectionViewCell, UICollectionViewDelegateFlowLayout
         return layoutAttributes
     }
     
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+    }
+    
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+extension SearchResultCell {
+
+    func fillInfo(for item: SearchItem, isLast: Bool) {
+        self.lastCell = isLast
+        self.title.text = item.name
+        self.author.text = "by \(item.owner?.login ?? "")"
+        self.subTitle.text = """
+        \(item.description ?? "")
+        """
+    }
+}
+
+private extension SearchResultCell {
+    
+    func setupUI() {
+        let arrangedSubviews = setupSubviews()
+        arrangedSubviews.forEach{contentView.addSubview($0)}
```

Comment:
Just code style - spaces between instructions enhance readability:
```suggestion
        arrangedSubviews.forEach{ contentView.addSubview($0) }
```

Reply:
will fix